clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -151,6 +153,45 @@ public HubConnectionState getConnectionState() {
         return connectionState;
     }
 
+
+    public CompletableFuture startAsync() throws Exception {
```

Comment:
I know this was originally done when prototyping, but if we're doing this for real now, we should call it `start` and replace the existing one. If that means updating the tests, let's get that done :)